src/Sarif.Multitool.FunctionalTests/TestData/DoNotUseFriendlyNameAsRuleId/Invalid_Expected.sarif

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
```

Comment:
Sorry to quibble on this, but is Invalid_Expected a good name? What would the file Invalid_Unexpected contain? This file shouldn't even exist because it expresses a runtime condition that we can't reliably provoke.

I am not sure I understand your complete vision for testing here. Maybe we could get on the phone to discuss just to be sure?

Reply:
@michaelcfanning I now remember the discussion we had on this point, where you explained why the _Expected files are not needed. All the test has to do is specify the expected number of results, and the JPointer locations of each result. I actually have a private branch where i started to do this but other work took priority.

That change, together with putting the input files into embedded resources for faster execution, would constitute my "vision for testing".

If you agree, no need for a call.

---
In reply to: [214739447](https://github.com/Microsoft/sarif-sdk/pull/1068#discussion_r214739447) [](ancestors = 214739447)